sphinx_click/ext.py

```diff
@@ -43,7 +43,7 @@ def _get_usage(ctx: click.Context) -> str:
     return formatter.getvalue().rstrip('\n')  # type: ignore
 
 
-def _get_help_record(opt: click.Option) -> ty.Tuple[str, str]:
+def _get_help_record(opt: click.Option, ctx: click.Context) -> ty.Tuple[str, str]:
```

Comment:
Can you pass `ctx` as the first argument instead, please. Ditto for the other modified functions.